efficiency/Makefile

```diff
@@ -35,10 +38,22 @@ src/efficiency.o: src/efficiency.cc src/efficiency.h src/sdl_types.h
 src/helper.o: src/helper.cc src/helper.h src/sdl_types.h
 	$(CC) $(T3T3EXTENSION) $(CFLAGS) $(EXTRACFLAGS) $< -c -o $@
 
+src/performance.o: src/performance.cc src/performance.h src/sdl_types.h
+	$(CC) $(T3T3EXTENSION) $(CFLAGS) $(EXTRACFLAGS) $< -c -o $@
+
+src/helper_v2.o: src/helper_v2.cc src/helper_v2.h src/sdl_types.h
+	$(CC) $(T3T3EXTENSION) $(CFLAGS) $(EXTRACFLAGS) $< -c -o $@
+
 rooutil:
 	$(MAKE) -C ../code/rooutil
 
 clean:
 	rm -f $(OBJECTS) $(EXE)
 
+summary: 
+	wget http://uaf-10.t2.ucsd.edu/~phchang/lst/summary.tar.gz && tar xf summary.tar.gz && rm summary.tar.gz
+
+compare: 
+	wget http://uaf-10.t2.ucsd.edu/~phchang/lst/compare.tar.gz && tar xf compare.tar.gz && rm compare.tar.gz
+
```

Comment:
What are these exactly?

Reply:
this is to download the web pages html files.
The way I do it is I generate .md files then i use pandoc to turn them into html.
but lnx7188 does not have pandoc installed.
So it can't generate on the fly, so I decided to save them in my area. so people can download them web format.
Given that this won't change very often, I thought it'd be OK.
and I didn't want to save a gazillion number of .html files.

I am not 100% satisfied with this honestly.

Reply:
Sure, makes sense.

Concerning the location of these files, would it work better to add them to our /data2 directory of both phi3 and lnx7188, so that we don't depend on uaf-10 being available (which is increasingly doubtful recently)? These are small, tarred files, so it should be fine.

Reply:
Put them in a separate repository that one has do a recursive checkout